content/300-guides/500-other/880-troubleshooting-orm/050-creating-bug-reports.mdx

```diff
@@ -120,6 +120,18 @@ Please include any information about your environment and setup. Specifically it
 - Which **version of Prisma** you use (run `prisma -v` to see your Prisma version)
 - Which **version of Node.js** you use (run `node -v` to see your Node.js version)
 
+You can use the [`prisma debug`](/reference/api-reference/command-reference#debug) <span class="api"></span> command to retrieve debugging information.
+
+<Admonition>
+
+The `prisma debug` command is available from version 5.6.0 and newer. If you're using an older version of Prisma, you can use this command by running:
+
+```terminal
+npx prisma@latest debug
+```
+
+</Admonition>
+
 Here's an example of what this could look like in your bug report:
```

Comment:
I think this is incorrect now.

Maybe just move the `prisma debug` mention below that and add another example of that if that would be helpful?